src/test/queue.spec.ts

```diff
@@ -0,0 +1,74 @@
+import * as sinon from "sinon";
```

Comment:
@bkendall 
Are you sure you want to use type script here?
This would be the first type script file in the repos I believe.